### PR TITLE
Update [Drawer] component (docs & examples)

### PR DESCRIPTION
**Pull request by manojleaton**
Fixes #BLUI-4365 .

#### Changes proposed in this Pull Request:

- Updated the kitchen sink component with Drawer component.
- Please verify Drawer component MD3 updates in this PR [DrawerMD3](https://github.com/etn-ccis/blui-react-native-component-library/pull/411)

#### Screenshots / Screen Recording (if applicable)
Screenshot 2023-10-20 at 5 48 13 PM

#### To Test:

- yarn start

#### Any specific feedback you are looking for?

-

### CODE REVIEW

**Review by surajeaton — COMMENTED** [review]
Please resolve the conflicts.

**Review by ektaghag-eaton — COMMENTED** [review]
Screenshot 2023-10-26 at 3 00 58 PM
As per design Drawer header height is 64px , here it is 60px(after changing text variant of subtitle font-size is not matching with design)

Inactive navItem icon color and chevron is not matching with figma design


**Current** 
Screenshot 2023-10-26 at 3 10 20 PM

**Expected**
Screenshot 2023-10-26 at 3 10 27 PM

Can you update example with drawer subheader, drawer footer component

**Comment by ArshdeepSingh3-eaton** [comment]
Updated the code 
Drawer header height is getting dynamically adjusted as per the header component height

File: md3-themes/MD3BluiLightTheme.ts [hunk 1]
```diff
@@ -48,11 +48,17 @@ export const MD3BluiLightTheme = {
...
+        surfaceContainer: '#E5ECF0',
+        surfaceContainerLow: '#F2F6F8',
+
         // @TODO verify neutral30
         neutralVariant: {
             neutral30: '#414E54',
             neutral100: '#FFFFFF',
         },
+        darkBlackVariant:{
+            darkBlack100: '182022',
```
Comment (ektaghag-eaton) [inline md3-themes/MD3BluiLightTheme.ts]: missing #

**Review by ektaghag-eaton — DISMISSED** [review]

**Comment by ektaghag-eaton** [comment]
Please resolve conflicts.

**Review by ektaghag-eaton — APPROVED** [review]

**Pull request merged**